libs/elements/cards/src/lib/elewa-news-card/elewa-news-card.component.html

```diff
@@ -0,0 +1,12 @@
+<div class="news-section">
+  <ul>
+    <h2 class="title">News</h2>
+      <div class="list-section">
+        <div class="content">
+          <p class="heading">Conversational learning</p>
+          <p class="description">Lorem ipsum dolor sit amet. Vel vitae enim non placeat suscipit qui galisum itaque qui voluptas quia in provident sunt.</p>
+        </div>
+        <button class="read-more">Learn More</button>
```

Comment:
this needs to be dynamic. The input of type newsItem that your component receives is what should be shown here.